app/templates/components/doi-summary.hbs

```diff
@@ -30,8 +30,35 @@
     <span class="text-{{model.state}}">{{titleize model.state}}</span>
   </div>
 {{/if}}
+{{#if (and isPerson (cannot "update user")) }}
+  <div class="metadata license">
+    <div class="metrics-counter">
+      {{#if hasMetrics}}
+          {{#if citations}}
+            <i class="metrics-icons" citations-test-badge> {{fa-icon "quote-right"}} {{ format-numbers citations }} {{  pluralize citations 'Citation' without-count=true}}</i>
+          {{else}}
+            <i class="metrics-icons" citations-test-badge>{{fa-icon "info-circle"}}  No citations were reported.</i>
+          {{/if}}
+          {{#if hasUsage }}
+            {{#if views}}
+              <i class="metrics-icons"> {{fa-icon "eye"}}  {{ format-numbers views }}  {{ pluralize views 'View' without-count=true }} </i>
+            {{/if}}
+            {{#if downloads}}
+              <i class="metrics-icons"> {{fa-icon "download"}}  {{ format-numbers downloads }}   {{ pluralize downloads 'Download' without-count=true }} </i>
+            {{/if}}
+          {{else}}
+            <i class="metrics-icons">{{fa-icon "info-circle"}}  No usage information was reported.</i>
+          {{/if}}
+      {{else}}
+        <i>{{fa-icon "info-circle"}}  No citations were reported. No usage information was reported.</i>
```

Comment:
Is there a way to remove some duplication in this code? And please look at whitespace.